src/main/resources/env-map.yml

```diff
@@ -1,8 +1,8 @@
 env_vars:
-  DB_HOST:               '.[][] | select(.name == "publicauth-db")             | .credentials.host'
-  DB_NAME:               '.[][] | select(.name == "publicauth-db")             | .credentials.name'
-  DB_PASSWORD:           '.[][] | select(.name == "publicauth-db")             | .credentials.password'
-  DB_USER:               '.[][] | select(.name == "publicauth-db")             | .credentials.username'
+  DB_HOST:               '.[][] | select(.name == "publicauth-secret-service")             | .credentials.db_host'
+  DB_NAME:               '.[][] | select(.name == "publicauth-secret-service")             | .credentials.db_name'
+  DB_PASSWORD:           '.[][] | select(.name == "publicauth-secret-service")             | .credentials.db_password'
+  DB_USER:               '.[][] | select(.name == "publicauth-secret-service")             | .credentials.db_user'
   DB_SSL_OPTION:         '.[][] | select(.name == "publicauth-db")             | .credentials.ssl_option // "ssl=true"'
```

Comment:
As Richard noted on the other PR`DB_SSL_OPTION` is now available from `publicauth-secret-service` so probably worth updating at the same time? Just checked and they're all there now
```
cf curl "/v2/user_provided_service_instances" | jq '.resources[] | select(.entity.name == "publicauth-secret-service")'
```